evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -194,23 +193,22 @@ def raise_zero_division(input_features):
         explain_predictions_best_worst(pipeline, pd.DataFrame({"a": range(15)}), pd.Series(range(15)))
 
 
-@pytest.mark.parametrize("input_features", [1, [1], "foo", pd.DataFrame()])
-def test_explain_predictions_value_errors(input_features):
+def test_explain_predictions_value_errors():
     with pytest.raises(ValueError, match="Parameter input_features must be a non-empty dataframe."):
-        explain_predictions(None, input_features)
+        explain_predictions(None, pd.DataFrame())
 
 
 def test_output_format_checked():
-    input_features, y_true = pd.DataFrame({"a": range(15)}), pd.Series(range(15))
+    input_features, y_true = pd.DataFrame(data=[range(15)]), pd.Series(range(15))
```

Comment:
`pd.DataFrame({"a": range(15)})` produces a DataFrame with one column, but updated since the error text says we want a DataFrame with one row.